scm/helper.go

```diff
@@ -84,8 +84,13 @@ func (opt GetOrgOptions) valid() bool {
 	return opt.ID != 0 || opt.Name != ""
 }
 
-func (r *Repository) valid() bool {
-	return r.Path != "" && r.Owner != ""
+func (opt UpdateEnrollmentOptions) valid() bool {
```

Comment:
In general, I'm not a fan of these `valid()` methods, as least if they are only used in one method.